requests/sessions.py

```diff
@@ -543,81 +551,91 @@ def request(self, method, url,
 
         return resp
 
-    def get(self, url, **kwargs):
-        r"""Sends a GET request. Returns :class:`Response` object.
+    def get(self, url, params=None, **kwargs):
```

Comment:
I decided to add the positional parameter here instead of deleting it in `request.api`.